docker/cortx-deploy/cortx-all/third-party-rpms.txt

```diff
@@ -2,6 +2,7 @@ crontabs  #cortx-utils
 haproxy22 #cortx-s3srever
 java-1.8.0-openjdk-headless
 jq #cortx-prvsnr
+kafka  #cortx-utils
```

Comment:
Kafka is removed as part of https://github.com/Seagate/cortx-re/pull/803

Reply:
Removed kafka package